cli/azd/pkg/infra/provisioning/bicep/bicep_provider_test.go

```diff
@@ -555,11 +555,25 @@ func prepareDestroyMocks(mockContext *mocks.MockContext) {
 	})
 
 	// Delete deployment
-	mockContext.CommandRunner.When(func(args exec.RunArgs, command string) bool {
-		return strings.Contains(command, "az deployment sub delete")
-	}).Respond(exec.RunResult{
-		Stdout: "",
-		Stderr: "",
+	mockPollingUrl := "https://url-to-poll.net/keep-deleting"
+	mockContext.HttpClient.When(func(request *http.Request) bool {
+		return request.Method == http.MethodDelete &&
+			strings.Contains(
+				request.URL.Path, "/subscriptions/SUBSCRIPTION_ID/providers/Microsoft.Resources/deployments/test-env")
+	}).RespondFn(func(request *http.Request) (*http.Response, error) {
+		response, err := mocks.CreateEmptyHttpResponse(request, 202)
+		response.Header.Add("Operation-Location", mockPollingUrl)
```

Comment:
They are using `Operation-Location` header name instead of standard `Location` header?  Seems weird but please double check.

Reply:
turns out both headers can be used for LROs!!  
But in this case, they use the `location` one.
The difference is that `Operation-location` will still require a body and a non `204` response.

For `location`, the last response can be `204` with no content (body)